x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/torq/torq.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { lazy } from 'react';
+import { ActionTypeModel, GenericValidationResult } from '../../../../types';
+import { TorqActionParams, TorqConfig, TorqSecrets } from '../types';
+import * as i18n from './translations';
+
+function replaceReferencesWithNumbers(body: string) {
+  return body.replace(/\{\{[.\w]+\}\}/gm, "42");
+}
```

Comment:
I don't quite get this, why do we need to replace the mustache variables with numbers?

Reply:
This way we can easily validate the JSON body. E.g.
```json
{
   "key": {{some_number}}
}
```
Is not a valid JSON because the moustache template is for a numeric value (no quotation marks). In order to allow it when validating, and considering the fact that a number is a valid drop-in replacement for any scalar value template, we replace all templated values with numbers for validation.
LMKWYT